src/GitHub.VisualStudio/GitHubPackage.cs

```diff
@@ -59,17 +57,20 @@ void LogVersionInformation()
             var menuService = (IMenuCommandService)(await GetServiceAsync(typeof(IMenuCommandService)));
```

Comment:
Same comment as I made to your other `InitializeMenus()` method.

Reply:
Oops, I remember you said. I'm sorry I missed these. 😊